tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs

```diff
@@ -39,7 +40,7 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
                 data.Add(PhpVersions.Php81Version, imageHelper.GetGitHubActionsBuildImage(ImageTestHelperConstants.GitHubActionsBuster), PhpVersions.ComposerVersion);
                 data.Add(PhpVersions.Php82Version, imageHelper.GetGitHubActionsBuildImage(ImageTestHelperConstants.GitHubActionsBuster), PhpVersions.ComposerVersion);
 
```

Comment:
Curious why we have `PhpVersions.ComposerVersion`? Isn't that the same as `PhpVersions.Composer20Version`?

Reply:
It's the same version, but `PhpVersions.ComposerVersion` represents the default composer version that's used -- let me see if I can easily rename this property to make it more clear